api/src/test/java/org/pmiops/workbench/cohorts/CohortFactoryTest.java

```diff
@@ -0,0 +1,118 @@
+package org.pmiops.workbench.cohorts;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.pmiops.workbench.db.model.Cohort;
+import org.pmiops.workbench.db.model.CohortReview;
+import org.pmiops.workbench.db.model.User;
+import org.pmiops.workbench.db.model.Workspace;
+import org.pmiops.workbench.model.ReviewStatus;
+
+import java.sql.Timestamp;
+import java.time.Clock;
+import java.util.Collections;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.doReturn;
+import static org.mockito.Mockito.mock;
+
+public class CohortFactoryTest {
```

Comment:
I do agree that this test+class is pretty trivial but the reason I wanted it out of `CohortsControllerTest` is because it seemed a bit too heavy and integration like for the kind of tests I needed. Do we want to build out a larger suite of unit tests?

Reply:
Our tests in general may need improvement - but testing at the level of the API controller I don't think is too heavy at all, this is the only place in this PR where you can probably fully test the method behavior. I'm not totally sure what you mean by larger test suite - but I don't think we need more unit tests at this lower level.